### PR TITLE
Utilize API endpoint to generate UUID and id

### PR DESCRIPTION
**Pull request by daelynj**
We can use our existing API endpoint to POST a new Player entity and
receive a UUID and ID to use for authentication and organization
purposes.

### CODE REVIEW

File: lib/middleware/websocket/create_player.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'httparty'
+
+module Websocket
+  class CreatePlayer
+    def call
+      HTTParty.post('http://localhost:3000/api/players')
```
Comment (jgayfer) [inline lib/middleware/websocket/create_player.rb]: I dislike the idea of making HTTP calls to ourselves. Do we even need an endpoint to create players if this is our path forward?

File: lib/middleware/websocket/client.rb [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
-    attr_reader :connection, :client_attributes
+    attr_reader :connection, :uuid, :id
     attr_accessor :position
 
     def initialize(connection:)
       @connection = connection
       @position = 0
+      player = CreatePlayer.new.call
+      @uuid = player['uuid']
+      @id = player['id']
```
Comment (jgayfer) [inline lib/middleware/websocket/client.rb]: I'd rather we didn't make new instance variables that can be easily derived, ie `player.uuid`. It needlessly hides where this information is coming from, which hurts readability.

File: lib/middleware/websocket/controller.rb [hunk 3]
```diff
@@ -30,7 +30,12 @@ def on_close(connection)
 
     private
 
-    def find_client(connection: connection)
+    def generate_client(connection:)
+      @clients << Client.new(connection: connection)
+      @clients[-1].generate_player
```
Comment (jgayfer) [inline lib/middleware/websocket/controller.rb]: We can do `@clients.last` instead

File: lib/middleware/websocket/create_player.rb (hunk 1 above)
Comment (daelynj) [reply lib/middleware/websocket/create_player.rb]: @jgayfer fixed.

File: lib/middleware/websocket/controller.rb (hunk 3 above)
Comment (daelynj) [reply lib/middleware/websocket/controller.rb]: @jgayfer fixed.

File: lib/middleware/websocket/client.rb (hunk 2 above)
Comment (daelynj) [reply lib/middleware/websocket/client.rb]: @jgayfer fixed.

**Pull request merged**